CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 # use cmake 3.12 or later
 cmake_minimum_required (VERSION 3.12)
 
-project(vlneval VERSION 0.0.1 LANGUAGES CXX)
+set(PROJ_VERSION 0.0.1)
 
-# if the CETBUILDTOOLS_VERSION environmental variable is defined, use cetbuildtools
-if(DEFINED ENV{CETBUILDTOOLS_VERSION})
-# using cetbuildtools to create a ups product
+# if the CETMODULES_VERSION environmental variable is defined, use cetmodules
+if(DEFINED ENV{CETMODULES_VERSION})
 
-    # cetbuildtools contains our cmake modules
-    # Note that find package will add cetbuildtools Modules to CMAKE_MODULE_PATH
-    find_package(cetbuildtools REQUIRED)
+    # when using cetmodules, find_package(cetmodules..) must come before project(...)
+    find_package(cetmodules 2.29.06 REQUIRED)
```

Comment:
May I ask you please to confirm whether the exact version specification `2.29.06` is required here as well?

Reply:
Note that this is not an _exact_ version, but a _minimum_ version (or range, see https://github.com/usert5432/vlneval/pull/2#discussion_r762022757). We usually do this because there is a feature we need in this and future versions, or a bug we wish to avoid in older versions.

Reply:
Ok, makes sense. I did not understood the cmake specification correctly. Thank you!